Some minor updates

doctordns opened the pull request:
This update is a revision to an earlier one that somehow broke! My bad I assume!
Changes:
1. noted that the article focuses on both the ARM and MSOnline Backup cmdlets
2. noted you may also need to update any older script to use the latest cmdlets.
3. Updated region names in some example to "WestUS" from 'West US'
4. Embolding a cmdlet name for better visibility and consistency.
5. Added details on how you can use PowerShell to download the MARS agent
6. Updated an example to be correct
7. Added details about the environment variable and that it's not updated.
8, Corrected example on setting pass codes to be correct - earlier version did nto reallhy work.
9. Updated outout of example to be up to date
10. Removed a bit of passive voice from the document

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ClarityQueen commented:
Edit complete. Sent this info to gary:
I am not seeing the need to cap “web service.” Also, usually we don’t cap the word “service,” for example DHCP Server service, so I am following that guidance for Batch Execution service and Request-Response service. I apologize in advance if styles have been established for these but not documented. I also question why Request-Response is hyphenated. Keep in mind that the S in the acronyms RRS and BES stands for “service,” so using “BES service” is redundant.

You are inconsistent with capping the term “trained model.” I think it is a generic term unless you are referring to the UI. Note that you need to change “Train model” to “Trained Model” in some of your graphics.

doctordns committed:
Some minor updates

This update is a revision to an earlier one that somehow broke! My bad I assume!
Changes:
1. noted that the article focuses on both the ARM and MSOnline Backup cmdlets
2. noted you may also need to update any older script to use the latest cmdlets.
3. Updated region names in some example to "WestUS" from 'West US'
4. Embolding a cmdlet name for better visibility and consistency.
5. Added details on how you can use PowerShell to download the MARS agent
6. Updated an example to be correct
7. Added details about the environment variable and that it's not updated.
8, Corrected example on setting pass codes to be correct - earlier version did nto reallhy work.
9. Updated outout of example to be up to date
10. Removed a bit of passive voice from the document
